modules/db/src/main/java/org/opencastproject/db/Activator.java

```diff
@@ -167,9 +167,9 @@ public void start(BundleContext bundleContext) throws Exception {
       throw new RuntimeException("Unable to verify SQL credentials have required permissions!", e);
     } finally {
       try {
-        statement.executeQuery("DROP TABLE " + tableName + ";");
+        statement.executeUpdate("DROP TABLE " + tableName + ";");
```

Comment:
It's out of the scope of this PR, but shouldn't we use PreparedStatements?

Reply:
In general, yes, absolutely! It should be okay in this special case, though.